content/docs/reference-dom-elements.md

```diff
@@ -73,70 +73,70 @@ const divStyle = {
+Stil anahtarları, JS'deki DOM elemanlarındaki özelliklere erişimle tutarlı olmak için camelCased'dir. (örneğin `node.style.backgroundImage`). Vendor ön ekleri [`ms`'ten hariç](http://www.andismith.com/blog/2012/02/modernizr-prefixed/) büyük harf ile başlamalıdır. Bundan dolayı yukarıda `WebkitTransition`, büyük harfle "W" başlamıştır.
 
-React will automatically append a "px" suffix to certain numeric inline style properties. If you want to use units other than "px", specify the value as a string with the desired unit. For example:
+React otomatik olarak "px" son ekini belirli stil özelliklerine eklemektedir. "px" haricindeki diğer birimleri kullanmak istiyorsanız, değeri string olarak birimiyle beraber yazmalısınız.
 
 ```js
-// Result style: '10px'
+// Sonuç: '10px'
 <div style={{ height: 10 }}>
-  Hello World!
+  Merhaba Dünya!
 </div>
 
-// Result style: '10%'
+// Sonuç: '10%'
 <div style={{ height: '10%' }}>
-  Hello World!
+  Merhaba Dünya!
 </div>
 ```
 
-Not all style properties are converted to pixel strings though. Certain ones remain unitless (eg `zoom`, `order`, `flex`). A complete list of unitless properties can be seen [here](https://github.com/facebook/react/blob/4131af3e4bf52f3a003537ec95a1655147c81270/src/renderers/dom/shared/CSSProperty.js#L15-L59).
+Tüm stil özellikleri piksel string'ine dönüştürülemez. Bazıları birimsiz kalmaktadır. Örneğin `zoom`, `order`, `flex`. Bütün birimi olmayan özellikler [burada](https://github.com/facebook/react/blob/4131af3e4bf52f3a003537ec95a1655147c81270/src/renderers/dom/shared/CSSProperty.js#L15-L59) görebilirsiniz.
 
 ### suppressContentEditableWarning {#suppresscontenteditablewarning}
 
-Normally, there is a warning when an element with children is also marked as `contentEditable`, because it won't work. This attribute suppresses that warning. Don't use this unless you are building a library like [Draft.js](https://facebook.github.io/draft-js/) that manages `contentEditable` manually.
+Normalde, alt elemanları bulunan bir eleman `contentEditable` olarak işaretlendiğinde çalışmayacağına dair uyarı oluşmaktadır. `suppressContentEditableWarning` niteliği bu uyarıyı bastıracaktır. Bu niteliği [Draft.js](https://facebook.github.io/draft-js/) gibi `contentEditable`'ı manuel olarak yöneten bir kütüphane yazmayacaksınız kullanmamanızı tavsiye ederiz.
 
 ### suppressHydrationWarning {#suppresshydrationwarning}
 
-If you use server-side React rendering, normally there is a warning when the server and the client render different content. However, in some rare cases, it is very hard or impossible to guarantee an exact match. For example, timestamps are expected to differ on the server and on the client.
+Sunucu üzerinden ekran oluşturma yapıyorsanız, sunucu ve istemcinin ekranı farklı oluşturduğuna dair uyarı alırsınız. Fakat, çok nadiren, tam eşleşme neredeyse imkansızdır. Örneğin, timestamps sunucu ve istemcide farklı olması beklenir.
```

Comment:
render kelimesini render olarak ele aldığımız için;
Sunucu üzerinden ekran oluşturma yapıyorsanız,
yerine
Sunucu tarafında render ediyorsanız,
daha uygundur.

Reply:
Done.